### PR TITLE
IA-907 Rename menu entries

### PR DESCRIPTION
**Pull request by SimonDus**
Rename old " List " menu entries to more detailed Form List, Org Units List etc....

## Self proof reading checklist

- [x] Did I use eslint and black formatters
- [x] Is my code clear enough and well documented
- [x] Are my typescript files well typed
- [x] Did I add translations

## How to test

Just check the menu in both lanagues

## Print screen / video

image

### CODE REVIEW

**Review by codeKameleon — APPROVED** [review]
tiny detail, maybe the first letter in lowercase for "List" ?

File: hat/assets/js/apps/Iaso/domains/app/translations/en.json [hunk 1]
```diff
@@ -406,6 +406,11 @@
     "iaso.label.linkOrgUnitReferenceSubmission": "Link OrgUnit to reference submission",
     "iaso.label.links": "Links",
     "iaso.label.list": "List",
+    "iaso.label.formList": "Form List",
+    "iaso.label.orgUnitList": "Org units List",
+    "iaso.label.dataSourceList": "Data Sources List",
+    "iaso.label.beneficiariesList": "Beneficiaries List",
+    "iaso.label.planningList": "Planning List",
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/app/translations/en.json]: Can you reorder the whole file by alphanum please ?

File: hat/assets/js/apps/Iaso/domains/app/translations/fr.json [hunk 2]
```diff
@@ -404,6 +404,11 @@
     "iaso.label.linkOrgUnitReferenceSubmission": "Lier l'unité d'organisation à sa soumission de definition",
     "iaso.label.links": "Liens",
     "iaso.label.list": "Liste",
+    "iaso.label.formList": "Liste de formulaires",
+    "iaso.label.orgUnitList": "Liste d'unité d'organisation",
+    "iaso.label.dataSourceList": "Liste des sources de données",
+    "iaso.label.beneficiariesList": "Liste des bénéficiaires",
+    "iaso.label.planningList": "Liste des plannings",
```
Comment (beygorghor) [inline hat/assets/js/apps/Iaso/domains/app/translations/fr.json]: same here

**Review by beygorghor — CHANGES_REQUESTED** [review]
Re-order translation file by alphanum please

**Comment by SimonDus** [comment]
> tiny detail, maybe the first letter in lowercase for "List" ?

Yep did it

**Review by beygorghor — APPROVED** [review]

**Pull request merged**